src/styles/libs/fonts.css

```diff
@@ -0,0 +1,19 @@
+@font-face {
+  font-family: Avenir;
+  src: url('../../fonts/Avenir-Book.ttf');
+  src:
+    local('../../fonts/Avenir-Book.ttf'),
+    url('../../fonts/Avenir-Book.ttf') format('truetype');
```

Comment:
fix formatting

Reply:
@mrkovaliv, please tell me in which way you want to see this. Because it is my prettier's setting. I need to manage them)

Reply:
sorry, everything is good, I saw uncorect formatting (may be some glitches on my computer)